### ISSUE TITLE
Sentiment does not exist in the docker image

### ISSUE THREAD

**Issue by lerra** [src-d/hercules#322]
Hi,

It looks like the docker image does not support the sentiment feature and I got the version 

`docker run --rm -v $(pwd):/io srcd/hercules hercules --sentiment --pb https://github.com/django/django
Error: unknown flag: --sentiment
Usage:
  hercules [flags]
  hercules [command]

<standard help menu>

unknown flag: --sentiment
`
Here is the version output
`docker run --rm -v $(pwd):/io srcd/hercules hercules version
Version: 10
Git:     5263b0a6fa8bd2a0af2246106d7b1b076aac75aa
`

**Comment by vmarkovtsev** [comment]
Yep, this is intended: including TensorFlow adds 300MB to the image size. Besides, it works much slower on CPU and including a GPU flavor is harder.

I can prepare a separate image though.

**Comment by lerra** [comment]
That would be really nice if that was available!

**Comment by lerra** [comment]
When do you think you will have time to prepare it?
